### PR TITLE
Merge completed README.md and crime_predictor.yaml

### PR DESCRIPTION
**Pull request by thomassiu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,82 @@
-# DSCI_522_Group24
\ No newline at end of file
+# Crime prediction in Vancouver
+
+- contributors: Francisco Mejia, Jasmine Ortega, Thomas Siu, Shi Yan Wang
+
+This is the data analysis project of group 24 (Cohort 6, 2022) for DSCI 522 (Data Science workflows); a course in the Master of Data Science program at the University of British Columbia.
```
Comment (jasmineortega) [inline README.md:5]: Testing out the comment feature.

**Review by jasmineortega — COMMENTED** [review]
Approved, very thorough!

File: README.md (hunk 1 above)
Comment (thomassiu) [reply README.md:5]: Closing this conversation as resolved.

**Comment by sy25wang** [comment]
Approved

**Pull request merged**